eks_infrastructure/build_param.json

```diff
@@ -1,13 +1,13 @@
 {
-    "operation": "delete",
+    "operation": "create",
```

Comment:
lets keep the default to `upgrade_nodegroup` operation so that we do not have to make an explicit change next time

Reply:
I had it as that before and then remembered that we need to create new clusters in private repo account and purposely changed to create.